libs/api/domains/national-registry-v3/src/lib/graphql/models/nationalRegistryAddress.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+
+@ObjectType('NationalRegistryV3Address')
+export class Address {
+  @Field(() => String, { nullable: true })
+  streetName?: string | null
+
+  @Field(() => String, { nullable: true })
+  postalCode?: string | null
+
+  @Field(() => String, { nullable: true })
+  city?: string | null
+
+  @Field(() => String, { nullable: true })
+  municipalityText?: string | null
+
+  @Field(() => String, { nullable: true })
+  municipalityCode?: string | null
+}
```

Comment:
I would really like to have a single national-registry domain and normalise (share) these object types as much as possible. It may take more work now, but it will make our schema nicer and I'm sure it will save us work and pain later when we transition all our applications to this new service.